pyControl4/blind.py

```diff
@@ -0,0 +1,117 @@
+"""Controls Control4 blind devices.
+"""
+
+
+class C4Blind:
+    def __init__(self, C4Director, item_id):
+        """Creates a Control4 Blind object.
+
+        Parameters:
+            `C4Director` - A `pyControl4.director.C4Director` object that corresponds to the Control4 Director that the blind is connected to.
+
+            `item_id` - The Control4 item ID of the blind.
+        """
+        self.director = C4Director
+        self.item_id = item_id
+
+    async def getBatteryLevel(self):
+        """Returns the battery of a blind. We currently don't know the range or meaning."""
```

Comment:
@brunchboy What does your blind report? Is there any way we can compare the battery level reported by Control4 with another source (ex. a mobile app provided by the blind manufacturer?) so we can find out what the range is?

Reply:
Unfortunately, the mobile app shows nothing but the current position of the blind. I had no idea there were even batteries in the blinds until I saw this variable in the output from Director with the help of your library, and I am rather unhappy to learn I probably have an overpriced replacement to look forward to sometime in my future! 😆 I had believed there was mains power routed to the blinds inside the walls. I assume this variable is something the dealers use for remote support.

Reply:
My blinds are currently reporting values around 50. I wonder if this is a percentage, but am not sure. When I finally get my new blinds installed in the rest of the windows I will compare the values they report when freshly installed; that might help us decide if it actually is. If they start around 100, it would be suggestive.

Reply:
That sounds good, just leave a comment on this PR when you get your new blinds!

Reply:
Will do! I can’t believe how long it is taking the dealer to put together a bid. But things are difficult all around with supply chains and labor thanks to CoViD. Once they are installed I will do this research and report back.

And I am glad you liked the PR. Careful documentation is a habit I definitely learned while doing other reverse-engineering projects… in fact, one of my repos’ main purposes is to pull together the [discoveries I’ve made](https://djl-analysis.deepsymmetry.org/djl-analysis) about the Pioneer DJ protocol. 😄